data/xml/E83.xml

```diff
@@ -39,70 +39,70 @@
       <url>E83-1005</url>
     </paper>
     <paper id="6">
-      <title>AN EXPERT SYSTEM FOR THE PRODUCTION OF PHONEME STRINGS FROM UNMARKED ENGLISH TEXT USING MACHINE-INDUCED RULES</title>
+      <title>An Expert System for the Production of Phoneme Strings From Unmarked <fixed-case>E</fixed-case>nglish Text Using Machine-Induced Rules</title>
       <author><first>Alberto Maria</first><last>Segre</last></author>
       <author><first>Bruce Arne</first><last>Sherwood</last></author>
       <author><first>Wayne B.</first><last>Dickerson</last></author>
       <url>E83-1006</url>
     </paper>
     <paper id="7">
-      <title>VOCAL INTERFACE FOR A MAN-MACHINE DIALOG</title>
+      <title>Vocal Interface for a Man-Machine Dialog</title>
       <author><first>Dominique</first><last>Beroule</last></author>
       <url>E83-1007</url>
     </paper>
     <paper id="8">
-      <title>KNOWLEDGE ENGINEERING APPROACH TO MORPHOLOGICAL ANALYSIS</title>
+      <title>Knowledge Engineering Approach to Morphological Analysis</title>
       <author><first>Harri</first><last>Jäppinen</last></author>
       <author><first>Aarno</first><last>Lehtola</last></author>
       <author><first>Esa</first><last>Nelimarkka</last></author>
       <author><first>Matti</first><last>Ylilammi</last></author>
       <url>E83-1008</url>
     </paper>
     <paper id="9">
-      <title>A PROLOG IMPLEMENTATION OF LEXICAL FUNCTIONAL GRAMMAR AS A BASE FOR A NATURAL LANGUAGE PROCESSING SYSTEM</title>
+      <title>A <fixed-case>P</fixed-case>rolog Implementation of Lexical Functional Grammar as a Base for a Natural Language Processing System</title>
       <author><first>Werner</first><last>Frey</last></author>
```

Comment:
Lexical Functional Grammar here, I think? Also maybe Natural Language Processing?

Reply:
Lexical Functional Grammar was a (common) case where I got lazy and often did not protect it. I am unsure whether it is consistently titlecased in running text. Could be my bias because Tree Adjoining Grammar is sometimes titlecased and sometimes not, and I prefer not to.